proxy/logging/LogAccess.cc

```diff
@@ -1315,6 +1315,61 @@ LogAccess::marshal_version_build_number(char *buf)
   -------------------------------------------------------------------------*/
 
 int
+LogAccess::marshal_proxy_protocol_version(char *buf)
+{
+  const char *version_str = nullptr;
+  int len                 = INK_MIN_ALIGN;
+
+  if (m_http_sm) {
+    NetVConnection::ProxyProtocolVersion ver = m_http_sm->t_state.pp_info.proxy_protocol_version;
+    switch (ver) {
+    case NetVConnection::ProxyProtocolVersion::V1:
+      version_str = "V1";
+      break;
+    case NetVConnection::ProxyProtocolVersion::V2:
+      version_str = "V2";
+      break;
+    case NetVConnection::ProxyProtocolVersion::UNDEFINED:
+    default:
+      version_str = "UNK";
+      break;
+    }
```

Comment:
This this be '-'? That would seem to follow the if something is not set, log `-`. Or is UNDEFINED different than not set?

Reply:
> This this be '-'? That would seem to follow the if something is not set, log `-`. Or is UNDEFINED different than not set?

Ah agree. `-` is indeed better.